src/api/Users.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @flow
+ * @file Helper for the box Users API
+ * @author Box
+ */
+
+import Base from './Base';
+
+/**
+ * Fields we want include:
+ *  -
+ */
+
+class Users extends Base {
+    /**
+     * API URL for Users
+     *
+     * @param {string} [id] - A box user id. Defaults to 'me' if empty.
+     * @return {string} base url for users
+     */
+    getUrl(id?: string): string {
+        const userId = id || 'me';
+        return `${this.getBaseApiUrl()}/users/${userId}?fields=login,avatar_url,name`;
```

Comment:
I have a change that makes fields programatic, i'll let you know when that's in. This is fine but in case we want to use the users endpoint for other stuff we should eventually make it more flexible.

Reply:
I dont think we should request avatar_url as according to the docs it is deprecated. Should just use the avatar endpoint

Reply:
^ if that, then is fields needed? default values suffice?